Source/missiles.cpp

```diff
@@ -2156,13 +2156,14 @@ void AddGolem(Missile &missile, const AddMissileParameter &parameter)
 
 	int playerId = missile._misource;
 	Player &player = Players[playerId];
+	Monster &monster = Monsters[playerId];

```

Comment:
Since we know more specificially
```suggestion
	Monster &golem = Monsters[playerId];
```

Reply:
Done